resources/views/auth/login.html.twig

```diff
@@ -15,7 +15,7 @@
         <section class="tc-auth" data-c-padding="tb(triple)" data-c-overlay="black(80)" style="background-image: url('/images/bg_crowd.jpg');">
             <div class="tc-auth-wrapper" data-c-padding="tb(triple)" data-c-container="small">
                 {# Return to the homepage link. #}
-                <a data-c-color="white" href="/" title="Return Home">
+                <a data-c-color="white" href={{ home_url }} title="Return Home">
```

Comment:
Might as well localize the `title` while you're in here 😉

Reply:
Missed that 👍